lms/static/js/demographics_collection/Wizard.jsx

```diff
@@ -37,19 +37,23 @@ export default class Wizard extends React.Component {
   }
 
   findSubComponentByType(type) {
-    return this.props.children.filter((child) => child.type.name === type)
+    return React.Children.map(this.props.children, child => {
+      if(child.type.name === type) {
+        return child;
+      }
+    })
```

Comment:
Won't this return a bunch of `undefined` items as well? Or does `Children.map` have filtering in it? E.g.
```
const children = ["April", "Benjamin", "Chelsie"
children.map(child => {if (child === "April") {return child;}})
> ["April", undefined, undefined]
```

Reply:
Good call. Although it works, changing it to `toArray.filter` from `.map` is cleaner.